docs/source/reference/numpysupported.rst

```diff
@@ -315,6 +315,7 @@ The following top-level functions are supported:
 * :func:`numpy.tril` (second argument ``k`` must be an integer)
 * :func:`numpy.triu` (second argument ``k`` must be an integer)
 * :func:`numpy.unique` (only the first argument)
+* :func:`numpy.repeat`
```

Comment:
Will need an update to declare which arguments are supported. Also, given this list is alphabetically ordered, needs to go up a bit.